BigFootVentures/BigFootVentures.Application.Web/Scripts/Views/Home/Brand.js

```diff
@@ -1,6 +1,7 @@
 ﻿$(function () {    
     InitScript();
-
+    $('#infoMaximize').hide();
```

Comment:
can we put this inside the InitScript?

Reply:
also please use name instead of ID, make it "Button.Info.Maximize", also do it on other selector